deployments/docker-compose.tezos.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.2"
+
+volumes:
+  node_data:
+  client_data:
+
+services:
+ tezosnode:
+    image: tezos/tezos:mainnet
+    hostname: tezosnode
+    command: tezos-node --net-addr :9732
+    ports:
+      - "9732:9732"
+      - "8732:8732"
+    expose:
+      - "8732"
+    volumes:
+      - node_data:/home/srv/tezos/node
+      - client_data:/home/srv/tezos/client
+    restart: on-failure
+    networks:
+      - scrapers-network
+
+ xtz:
+    build:
+      context: ../../../..
+      context: $GOPATH
+      dockerfile: $GOPATH/src/github.com/diadata-org/diadata/build/Dockerfile-xtz
+    image: ${DOCKER_HUB_LOGIN}/xtz
+    networks:
```

Comment:
should be:
 xtz:
    build:
      context: ../../../..
      dockerfile: $GOPATH/src/github.com/diadata-org/diadata/build/Dockerfile-xtz
    image: ${DOCKER_HUB_LOGIN}/${STACKNAME}_xtz

Reply:
updated